griptape/structures/structure.py

```diff
@@ -115,18 +122,72 @@ def output_task(self) -> Optional[BaseTask]:
     def finished_tasks(self) -> list[BaseTask]:
         return [s for s in self.tasks if s.is_finished()]
 
+    @property
+    def default_config(self) -> BaseStructureConfig:
+        config = OpenAiStructureConfig()
+
+        if self.prompt_driver is not None:
```

Comment:
We should use a more pythonic `@deprecated` decorator for those things:

```python
@property
@deprecated("use `config.global_drivers.prompt_driver` instead.")
def prompt_driver(self) -> Optional[BasePromptDriver]:
    return None
```

Reply:
Added a `deprecated` decorator and refactored.